kotsadm/pkg/apiserver/server.go

```diff
@@ -130,6 +130,7 @@ func Start() {
 	r.Path("/api/v1/app/{appSlug}/registry").Methods("OPTIONS", "GET").HandlerFunc(handlers.GetAppRegistry)
 	r.Path("/api/v1/app/{appSlug}/registry/validate").Methods("OPTIONS", "POST").HandlerFunc(handlers.ValidateAppRegistry)
 	r.Path("/api/v1/app/{appSlug}/config").Methods("OPTIONS", "PUT").HandlerFunc(handlers.UpdateAppConfig)
+	r.Path("/api/v1/app/{appSlug}/liveConfig").Methods("OPTIONS", "PUT").HandlerFunc(handlers.LiveAppConfig)
```

Comment:
if you wanna keep the pattern alive :)
```suggestion
	r.Path("/api/v1/app/{appSlug}/liveconfig").Methods("OPTIONS", "PUT").HandlerFunc(handlers.LiveAppConfig)
```

Reply:
would it make more sense to use `GET` here? since this doesn't update anything

Reply:
also probably true

Reply:
"why not use GET":
`TypeError: Failed to execute 'fetch' on 'Window': Request with GET/HEAD method cannot have body.`